presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1634,4 +1644,31 @@ protocol::NodeStatus PrestoServer::fetchNodeStatus() {
   return nodeStatus;
 }
 
+void PrestoServer::registerDynamicFunctions() {
+  std::error_code
+      ec; // For using the non-throwing overloads of functions below.
```

Comment:
// For using the non-throwing overloads of functions below.
std::error_code ec;